test/integration/running/test_clean.py

```diff
@@ -1,36 +1,47 @@
 import os
 
-from utils import get_instance_id, get_stateboard_name
-from utils import DEFAULT_CFG, DEFAULT_DATA_DIR, DEFAULT_LOG_DIR, DEFAULT_RUN_DIR
 from utils import write_conf
 from utils import check_instances_running
 from utils import check_instances_stopped
 
 
-CARTRIDGE_CONF = '.cartridge.yml'
-
 FILES_TO_BE_DELETED = ['log', 'workdir', 'console-sock', 'notify-sock', 'pid']
 
 
-def get_instance_files(project, instance_id, log_dir=DEFAULT_LOG_DIR, data_dir=DEFAULT_DATA_DIR,
-                       run_dir=DEFAULT_RUN_DIR):
+def get_instance_files(project, instance_name, log_dir=None, data_dir=None, run_dir=None):
+    run_dir = project.get_run_dir(run_dir)
+
+    return {
+        'log': project.get_log_dir(instance_name, log_dir),
+        'workdir': project.get_workdir(instance_name, data_dir),
+        'console-sock': project.get_console_sock(instance_name, run_dir),
+        'pid': project.get_pidfile(instance_name, run_dir),
+        'notify-sock': project.get_notify_sock(instance_name, run_dir),
+    }
+
+
+def get_stateboard_files(project, log_dir=None, data_dir=None, run_dir=None):
+    run_dir = project.get_run_dir(run_dir)
+
     return {
-        'log': os.path.join(project.path, log_dir, '%s.log' % instance_id),
-        'workdir': os.path.join(project.path, data_dir, instance_id),
-        'console-sock': os.path.join(project.path, run_dir, '%s.control' % instance_id),
-        'pid': os.path.join(project.path, run_dir, '%s.pid' % instance_id),
-        'notify-sock': os.path.join(project.path, run_dir, '%s.notify' % instance_id),
+        'log': project.get_sb_log_dir(log_dir),
+        'workdir': project.get_sb_workdir(data_dir),
+        'console-sock': project.get_sb_console_sock(run_dir),
+        'pid': project.get_sb_pidfile(run_dir),
+        'notify-sock': project.get_sb_notify_sock(run_dir),
     }
 
 
-def create_instances_files(project, instance_names=[],
-                           log_dir=DEFAULT_LOG_DIR, data_dir=DEFAULT_DATA_DIR, run_dir=DEFAULT_RUN_DIR):
-    instance_ids = [get_instance_id(project.name, instance_name) for instance_name in instance_names]
-    instance_ids.append(get_stateboard_name(project.name))
+def create_instances_files(project, instance_names=[], stateboard=False, log_dir=None, data_dir=None, run_dir=None):
+    all_files = [
```

Comment:
Passing a large number of arguments in this form is not very convenient and is difficult to scale. I suggest using ``** kwargs``  with the ``.setdefault`` method.